### ISSUE TITLE
Problem implementing the api

### ISSUE THREAD

**Issue by unet2581** [abdulmueid/mpesa-php-api#10]
Hello
First I want to thank you for creating this api. I am having problems implementing the api, I am using PuTTy  (on Windows) to log in to my server and use composer.   I managed to install the api but I am stack on the first step "Load the configuration from file.", I get the error  -jailshell: syntax error near unexpected token `(' .

**Comment by abdulmueid** [comment]
Looks like a syntax error. Check the file again to make sure it's properly formatted and not missing any parenthesis or semi colons.

**Comment by unet2581** [comment]
HelloThank you for your reply.  To avoid any mistakes I copied the command, and I pasted the following:config = \abdulmueid\mpesa\Config::loadFromFile('/abdulmueid/mpesa/config.php');
I made a copy of the file"config.example.php" on the mpesa folder and renamed "config.php".  Do I need to add or edit the file? Like put the public key and the other fields. 
Joaquim Mussassa
https://urbannet.urbantutorial.com 

    On Thursday, September 10, 2020, 01:30:23 PM GMT+2, Abdul Mueid Akhtar <notifications@github.com> wrote:  
 
 


Looks like a syntax error. Check the file again to make sure it's properly formatted and not missing any parenthesis or semi colons.

**Comment by kishannareshpal** [comment]
Yes @unet2581, you will need to edit the config file with your own values. Also make sure you are passing the right path to `Config::loadFromFile()`, that is, relative to the working directory.

If you are having trouble finding out your configuration file path, you can instantiate your config method instead of loading from file as shown below:
```
$config = Config($public_key, $api_host, $api_key, $origin, $service_provider_code, $initiator_identifier, $security_credential);
```

;)

**Comment by unet2581** [comment]
> Yes @unet2581, you will need to edit the config file with your own values. Also make sure you are passing the right path to `Config::loadFromFile()`, that is, relative to the working directory.
> 
> If you are having trouble finding out your configuration file path, you can instantiate your config method instead of loading from file as shown below:
> 
> ```
> $config = Config($public_key, $api_host, $api_key, $origin, $service_provider_code, $initiator_identifier, $security_credential);
> ```
> 
> ;)

Thanks for your input... I will make sure I edit the config file.

**Comment by unet2581** [comment]
> Yes @unet2581, you will need to edit the config file with your own values. Also make sure you are passing the right path to `Config::loadFromFile()`, that is, relative to the working directory.
> 
> If you are having trouble finding out your configuration file path, you can instantiate your config method instead of loading from file as shown below:
> 
> ```
> $config = Config($public_key, $api_host, $api_key, $origin, $service_provider_code, $initiator_identifier, $security_credential);
> ```
> 
> ;)

Hello.
I was wondering if you could help me clarify some things.  I am editing the config.php file and I have some questions.
First can I use the testing api key in the field "api key"?
Second what do I put in the fields: 

- 'origin' => '',
   
-  'service_provider_code' => '',
 
-  'initiator_identifier' => '',
  
-  'security_credential' => ''

**Comment by unet2581** [comment]
> Looks like a syntax error. Check the file again to make sure it's properly formatted and not missing any parenthesis or semi colons.

Hello. Thanks for your input.
I think the error happened because my config.php was not edited.  I was wondering if you could help me clarify some things. I am editing the config.php file and I have some questions.
First can I use the testing api key in the field "api key"?
Second what do I put in the fields:

'origin' => '',

'service_provider_code' => '',

'initiator_identifier' => '',

'security_credential' => ''
